---
title: "Project"
editor: source
toc: false
format:
  html: default
  #docx: default
---

![](eOMIX_logo_long.jpg){fig-alt="e-OMIX logo" fig-align="center" width="50%"}

## The state of omics research

Over the past two decades, the life science community has witnessed a significant rise in "omics" research, specialized branches of biology dedicated to characterizing and quantifying an extensive spectrum of biologically significant molecules. Genomics, which offers an insight into an individual's complete DNA sequence, emerged as a pioneering molecular tool, was probably the first of these disciplines to have been developed and gained widespread recognition following the landmark achievement of the Human Genome Project in 2003. Comparable techniques were swiftly developed to quantify other molecular contents, such as gene variants, RNA transcripts, epigenomic markers, proteins, and metabolic by-products.

The field is still in constant mutation and is a source of continuous innovation for biological, clinical, and pharmaceutical research. Initially, omics methods were constrained to the analysis of entire tissue samples, known as "bulk omics." However, recent technological advancements have enabled a more refined approach, permitting the analysis of specific cell types, a domain known as "single-cell omics."

Concurrently with these technical advancements, the field of computational biology has undergone significant expansion. It has established a repertoire of statistical tools tailored to both generate and rigorously test hypotheses, utilizing the vast reservoir of data generated by omics analyses. Together, omics-based quantification methods and bioinformatics algorithms have paved the way to great progresses in the understanding of biological processes, and, in fine, the improvement of human health.

## Addressing Challenges in Integrative Omics

While a multitude of single omics pipelines are readily available and are used to answer experimental hypotheses through specific molecular lenses, combining multiple type of omics would considerably broaden the scope of investigations: the deployment of bioinformatic platforms where omics-specific pipelines converge to build centralized, standardized, and accessible datasets would be a tremendous improvement.

Such an endeavor however comes with significant hurdles. An example of these obstacles is the heterogeneity of molecular biology data. Each instrument is often associated with proprietary formats, hindering the sharing of analysis between research groups. Additional specifications might result from experimental workflow, the type of samples or other constraints. One of the objectives of our data treatment pipelines would then be to standardize and transform proprietary data into open-source formats, as well as resolve disparities among data types. In that perspective, e-OMIX will adhere to HL7's FHIR standards, ensuring interoperability of biomedical data, and make its own resources available for the scientific community, facilitating exchanges and collaboration.

In subsequent development steps, the platform will include scalable semi-automated statistical analyses to identify pertinent networks, hubs, and links between cellular constituents. Finally, e-OMIX will be able to incorporate clinical data as well, providing enhanced insights into both normal and pathological phenotypes. As an integrated tool, e-OMIX could then be leveraged to advance our understanding of cell biology, identify new biomarkers, and facilitate therapeutic developments.

\

## Funding

e-OMIX is a **WAL-IMAGIN** project and is supported by the **European Regional Development Fund (ERDF)**
